ui/app/common/attributeTypes/views/attributeInformation.html

```diff
@@ -26,23 +27,32 @@
                    ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}" ng-blur="::handleUpdate(attribute.name)">
 
             <input data-ng-switch-default ng-if="attribute.name == 'PRIMARY_CONTACT_NUMBER_1'"
-                   type="text" pattern-validate id="{{::attribute.name}}"
+                   type="text" pattern-validate id="{{::attribute.name}}" only-numbers-input="{{regexNumbers}}" with=""
                    ng-model="targetModel[attribute.name]" data-item-type="personAttribute" ng-init="targetModel[attribute.name]=targetModel[attribute.name] || '+258 '"
                    ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}" ng-blur="::handleUpdate(attribute.name)">
 
             <input data-ng-switch-default ng-if="attribute.name == 'ALTERNATIVE_CONTACT_NUMBER_1'"
-                   type="text" pattern-validate id="{{::attribute.name}}"
+                   type="text" pattern-validate id="{{::attribute.name}}" only-numbers-input="{{regexNumbers}}" with=""
                    ng-model="targetModel[attribute.name]" data-item-type="personAttribute" ng-init="targetModel[attribute.name]=targetModel[attribute.name] || '+258 '"
                    ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}" ng-blur="::handleUpdate(attribute.name)">
 
             <input data-ng-switch-default ng-if="attribute.name == 'ALTERNATIVE_CONTACT_NUMBER_2'"
-                   type="text" pattern-validate id="{{::attribute.name}}"
+                   type="text" pattern-validate id="{{::attribute.name}}" only-numbers-input="{{regexNumbers}}" with=""
                    ng-model="targetModel[attribute.name]" data-item-type="personAttribute" ng-init="targetModel[attribute.name]=targetModel[attribute.name] || '+258 '"
                    ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}" ng-blur="::handleUpdate(attribute.name)">
 
+            <input data-ng-switch-default ng-if="attribute.name == 'FATHER_NAME'"
```

Comment:
Why are we adding these two fields?

the scope of the ticket is just for validations